exporter/lokiexporter/README.md

```diff
@@ -78,26 +69,19 @@ attribute as label and the `resource` processor to give a hint to the Loki expor
 processors:
   attributes:
     actions:
-      - action: insert
-        key: event_domain
-        from_attribute: event.domain
       - action: insert
         key: loki.attribute.labels
-        value: event_domain
+        value: event.domain
 
   resource:
     attributes:
-      - action: insert
-        key: service_name
-        from_attribute: service.name
       - action: insert
         key: loki.resource.labels
-        value: service_name
+        value: service.name
 ```
 
-Currently, Loki does not support labels with dots. 
-That’s why to add Loki label based on `event.domain` OTLP attribute we need to specify two actions. The first one inserts a new attribute `event_domain` from the OTLP attribute `event.domain`. The second one is a hint for Loki, specifying that the `event_domain` attribute should be placed as a Loki label.
-The same approach is applicable to placing Loki labels from resource attribute `service.name`.
+Currently, Loki does not support label names with dots. 
+That's why lokiexporter normalizes label names to follow Prometheus label names standard before sending requests to Loki.
```

Comment:
We can point to `pkg/translator/prometheus/README.md#Labels` for more information on labels normalization.

Reply:
added one more sentence with a link to `pkg/translator/prometheus/README.md#Labels`